polaris-tokens/token-groups/color.json

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
For now, I suggest keeping this `color.light.json` so it's clear these are the `light` color scheme tokens in consuming code. While we don't officially support dark mode we do leverage dark color schemes throughout Polaris and Web. Until we establish next steps for light, dark, and beyond color schemes I think our naming should be very explicit so it's clear how tokens are being used and where to improve.